src/01-functions.js

```diff
@@ -9,7 +9,14 @@
  *  alwaysTrue();
  *  //> true
  */
-function alwaysTrue() {}
+function alwaysTrue() {
+  let a = true
+  return a;
```

Comment:
another way to do it, is to simply return `true` instead of storing it in a variable. Either is correct tho!